docs/source/general/settings.rst

```diff
@@ -151,9 +151,30 @@ with other freeradius modules.
 
 Set this to ``True`` if you are performing authorization exclusively through the REST API.
 
-Batch mail settings
-~~~~~~~~~~~~~~~~~~~
+Sending emails to users
+-----------------------
 
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT <importing_users.html#django-freeradius-batch-mail-subject>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE <importing_users.html#django-freeradius-batch-mail-message>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SENDER <importing_users.html#django-freeradius-batch-mail-sender>`_
+Emails can be sent to users whose usernames or passwords have been autogenerated. The content of these emails can be customized with the settings explained below.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``""``
+
+It is the subject of the mail to be sent to the users. Eg: ``Login Credentials``.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``""``
+
+The message should be a string in the format ``Your username is {} and password is {}``.
+
+The text could be anything but should have the format string operator ``{}`` for ``.format`` operations to work.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SENDER``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is the sender email which is also to be configured in the SMTP settings.
```

Comment:
specify the default value here as well (the actual default mentioned in the code), why should this setting be different from the others?